test/validation/api/ipsec/ipsec.c

```diff
@@ -765,6 +813,44 @@ static int ipsec_process_in(const ipsec_test_part *part,
 	return num_out;
 }
 
+static int ipsec_check_sa_expiry(enum ipsec_test_sa_expiry sa_expiry,
+				 odp_ipsec_packet_result_t *result)
+{
+	if (sa_expiry != IPSEC_TEST_EXPIRY_IGNORED &&
```

Comment:
The test whether sa_expiry equals IPSEC_TEST_EXPIRY_IGNORED could be done in the beginning to avoid repeating it later and cluttering the code. i.e.:

```
if (sa_expiry == IPSEC_TEST_EXPIRY_IGNORED)
        return 0;
```

Reply:
Ack. Will update so with the next version.